### PR TITLE
Feat: Access page backend and frontend

### PR DESCRIPTION
**Pull request by AJSahagun**

### CODE REVIEW

File: frontend/src/pages/access/hooks/useBulkRequest.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        await db.occupants.bulkDelete(idsToDelete);
+      }
+    }
+    console.log("up to date")
+  };
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      sendBulkRequest();
+    }, 60*30* 1000); // Every 1 hour
```
Comment (marius-hernandez) [inline frontend/src/pages/access/hooks/useBulkRequest.ts:36]: final na every 30 mins ang bulk request ng history?

**Review by marius-hernandez — CHANGES_REQUESTED** [review]
LGTM

**Comment by AJSahagun** [comment]
Check reply

**Review by marius-hernandez — APPROVED** [review]
LGTM

**Review by gonzalestrisha — APPROVED** [review]
good

**Pull request merged**